### DISCUSSION TITLE
Multiple clickable stuff

### DISCUSSION THREAD

**Discussion by iamnoksio** [Zrips/CMI#7506] [Enhancement]
[PLAYER] Noksio: Hello
Make the PLAYER rank display a different text than the player name(Noksio)

**Comment by MrCrazys** [comment]
> [PLAYER] Noksio: Hello Make the PLAYER rank display a different text than the player name(Noksio)

I honestly like this idea. It would be cool to be able to use CMI's raw message format for full control over the interactive elements of our chat. Perhaps the format could be something like `<T>[PLAYER]</T><H>Hover text</H><Next><T> Noksio:</T><H>Player information here.</H><SC>/msg Noksio</SC><Next>: Hello`.
